src/Schema/Schema.php

```diff
@@ -313,7 +351,7 @@ public function refreshTableSchema(string $name): void
      */
     public function getLastInsertID(string $sequenceName = ''): string
     {
-        if ($this->db->isActive()) {
+        if ($this->db->isActive() && $this->db->getPDO() !== null) {
```

Comment:
Looks like this check should be part of `isActive()`.

Reply:
done.